src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -188,10 +192,11 @@ export default class UniversalDiscoveryModule extends Component {
 
     onItemSelect(contentMeta) {
         const isLocationAllowed = !this.props.cotfAllowedLocations.length || this.props.cotfAllowedLocations.includes(contentMeta.id);
+        const contentMetaWitchContentTypeInfo = this.addContentTypeInfo([contentMeta])[0];
```

Comment:
Because of your change you're doing a kind of strange thing here. Currently, I don't see any value in changing the way of handling from a single item to an array of items in the `addContentTypeInfo` method